main-app/app/scripts/Gameboard/Controller/GameboardController.js

```diff
@@ -1,12 +1,13 @@
 (function () {
     'use strict';
     angular.module('Tombola.NoughtAndCrosses.gameboard')
-    .controller('GameboardController',function ($scope, Proxy, playerToggle, GameModel){
+    .controller('GameboardController',function ($scope, Proxy, playerToggle, GameModel, winStates){
 
         this.gameModel = GameModel;
+        this.players = playerToggle;
 
         this.gameboardTapped = function (gridNumberFromTable) {
-            if (GameModel.gameboard.charAt(gridNumberFromTable) !== '0' || GameModel.currentState === 'Win') {
+            if (GameModel.isSquareSelected(gridNumberFromTable) || GameModel.isGameInPlay()){
```

Comment:
Move this condition evaluation to the game model: method something like GameModel.canMakeMove(squareName)